src/main/java/net/dv8tion/jda/api/EmbedBuilder.java

```diff
@@ -750,7 +750,7 @@ public EmbedBuilder addField(@Nullable MessageEmbed.Field field)
      * @return the builder after the field has been added
      */
     @Nonnull
-    public EmbedBuilder addField(@Nullable String name, @Nullable String value, boolean inline)
+    public EmbedBuilder addField(@Nonnull String name, @Nonnull String value, boolean inline)
```

Comment:
You need to actually add null checks for these too. And please also update the exception docs

Reply:
Will do so.